MAUI/DataGrid/Column-types.md

```diff
@@ -0,0 +1,916 @@
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:DataGridTemplateColumn="CustomerID">
+    <syncfusion:DataGridTemplateColumn.CellTemplate>
+        <DataTemplate>
+            <Label Text="{Binding CustomerID}" TextColor="Blue" 
+                   XAlign="Center" YAlign="Center" />
+        </DataTemplate>
+    </syncfusion:DataGridTemplateColumn.CellTemplate>
+</syncfusion:DataGridTemplateColumn> 
+{% endhighlight %}
+{% highlight c# %}
+DataGridTemplateColumn templateColumn = new DataGridTemplateColumn()
+{
+    MappingName = "CustomerID",
+    Width = 50,
+};
+var dataTemplate = new DataTemplate(() =>
+{
+var label = new Label()
+{
+    TextColor = Color.Blue,
+    VerticalOptions = LayoutOptions.Center,
+    HorizontalOptions = LayoutOptions.Center
+};
+label.SetBinding(Label.TextProperty, "CustomerID");
+return label;
+});
+templateColumn.CellTemplate = dataTemplate;
+{% endhighlight %}
+{% endtabs %}
+
+The following code example illustrates how template column can be used to load a stock cell inside it:
+
+{% highlight xaml %}
+<ContentPage.Resources>
+    <ResourceDictionary>
+        <local:ImageConverter x:Key="imageConverter" /> 
+    </ResourceDictionary>
+</ContentPage.Resources>
+
+<ContentPage.ContentView>
+    <syncfusion:SfDataGrid x:Name="dataGrid"
+                           ColumnSizer="Star">
+
+        <syncfusion:SfDataGrid.Columns>
+            <syncfusion:DataGridTemplateColumn="Stock Change"
+                                           MappingName="StockChange">
+                <syncfusion:DataGridTemplateColumn.CellTemplate>
+                    <DataTemplate>
+                        <Grid>
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="*" />
+                            </Grid.ColumnDefinitions>
+                            <Image Grid.Column="0"
+                                   Source="{Binding StockChange,
+                                   Converter={StaticResource imageConverter}}" />
+                            <Label x:Name="changeValue" Grid.Column="1"
+                                   Text="{Binding StockChange}" TextColor="Black"
+                                   XAlign="Center" YAlign="Center">
+                            </Label>
+                        </Grid>
+                    </DataTemplate>
+                </syncfusion:DataGridTemplateColumn.CellTemplate>
+            </syncfusion:DataGridTemplateColumn>
+        </syncfusion:SfDataGrid.Columns>
+    </syncfusion:SfDataGrid>
+</ContentPage.ContentView>
+{% endhighlight %}
+
+In order to get the above code example working, write a converter to load images inside the `GridCell` based on the CellValue. The images that have to be loaded in the `GridCell` must be added as EmbeddedResource.
```

Comment:
In order to get the code example above working, write a converter to load images inside the `GridCell` based on the CellValue. The images that have to be loaded in the `GridCell` must be added as EmbeddedResource.